modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,146 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker_{context}"]
-= Creating a broker manually
+= Creating a broker as a developer
 
-To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+.Procedure
 
-.Prerequisites
-* Knative Eventing installed, which includes the `ClusterRole`.
+. If you do not have an existing event sink created to consume events from the broker, create a simple Knative Service to use as an event sink.
+.. Save the following YAML in a `.yaml` file:
++
 
-.Procedure
-. Create the `ServiceAccount` objects:
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display
+----
+
+.. Create the Service by applying the file:
++
+
+[source,terminal]
+----
+$ oc apply -f <FILENAME>
+----
+
+. If you do not have an existing event source, create an event source that emits events to the `default` broker. The following example shows how you can create a basic PingSource.
+.. Save the following YAML in a `.yaml` file:
 +
+
+[source,yml]
 ----
-$ oc -n default create serviceaccount eventing-broker-ingress
-$ oc -n default create serviceaccount eventing-broker-filter
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: ping-source
+spec:
+  schedule: "*/1 * * * *"
+  jsonData: '{"message": "Hello serverless!"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1beta1
+      kind: Broker
+      name: default
 ----
 
-. Give those objects RBAC permissions:
 +
+..  Create the event source by applying the file:
+
++
+[source,terminal]
 ----
-$ oc -n default create rolebinding eventing-broker-ingress \
-  --clusterrole=eventing-broker-ingress \
-  --serviceaccount=default:eventing-broker-ingress
-$ oc -n default create rolebinding eventing-broker-filter \
-  --clusterrole=eventing-broker-filter \
-  --serviceaccount=default:eventing-broker-filter
+$ oc apply -f <FILENAME>
 ----
 
-. Create the broker:
+. Create the broker by creating a Trigger object that uses the `knative-eventing-injection: enabled` annotation.
```

Comment:
It looks like you've swapped the rest of the instances of "trigger" to lowercase.

Reply:
It's because this is referring to the specific API Object rather than triggers as a general concept. I know we're still trying to figure this out even with things like Pod vs pod, but do you have any recommendations for this specific case? Should I just keep it lowercase everywhere? Thanks for all the comments!